pages/trial-procedures.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div id="main-content" class="text-wrapper w-full">
+    <h1 class="text-center mb-6">
+      Witchcraft trial procedure and evidence in Scotland
+    </h1>
+    <div>
+      <div class="lg:float-right lg:w-1/2 lg:ml-6 w-3/4 mx-auto">
+        <img src="../public/images/trial.png" class="w-full" />
+        <p class="text-xs text-center mx-auto">
+          Figure 1. Suspected witches from North Berwick appear before James VI
+          and two Scottish magistrates, from the English pamphlet Newes from
+          Scotland (1591) (Public domain, via Wikimedia Commons)
+        </p>
+      </div>
+      <p class="mb-3 text-justify">
+        Accused witches were tried in criminal courts. In the courtroom on the
+        day of the trial, there was a judge who presided over the proceedings
+        and made rulings on the law, a jury (fifteen property owning members of
+        the local community) who decided on a verdict of innocent or guilty, and
+        a clerk to keep records. There would also be a prosecutor to present the
+        accusations. The accused person would be able to speak in their own
+        defence. If the trial was held in the central Edinburgh court of
+        justiciary, the court would also assign a defence lawyer to the person.
+        If the jury convicted the accused person, the judge would pass sentence.
+        In a witchcraft case the sentence was usually death by strangulation
+        followed by burning. Convictions were common but not universal; some
+        people were acquitted.
+      </p>
+    </div>
+    <p class="mb-3 text-justify">
+      Evidence against the accused person had to be gathered beforehand. This
+      was often done by local church courts - kirk sessions or presbyteries.
+      These bodies were not criminal courts, but they could summon people and
+      interrogate them. Two main kinds of evidence were used against witchcraft
+      suspects: neighbours' testimony, and the suspect's own confession. If the
+      suspect was thought to have bewitched her or his neighbours, these
+      neighbours could make statements about this, telling stories about
+      quarrels and bewitchments that had followed. Or, if the suspect was
+      thought to have made a pact with the Devil, the suspect herself or himself
+      could be interrogated about this. Interrogations were often done
+      coercively; usually little was written down about this, but the most
+      common form of torture seems to have been sleep deprivation. This was used
+      due to its effectiveness and was usually conducted by local authorities.
+      Confession evidence tended to have much more about the suspect's dealings
+      with the Devil. Confessions could also implicate other suspects, if a
+      suspect was asked something like 'Who else was present when you met the
+      Devil?'.
+    </p>
+    <div>
+      <div class="lg:float-right lg:w-1/2 lg:ml-6 w-3/4 mx-auto">
```

Comment:
To mix things up, we could perhaps have this image float left instead of right. It might make the page feel a little more interesting visually.

Is there anywhere obvious to add sections to this as well? It is a lot of text to consume in one go so might benefit from sections being added

Reply:
done, looks good moved to left

Reply:
have added subheads, think this does make it easier to read

Reply:
Thanks. This looks good to me. Cheers for making the changes.